modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -380,6 +381,7 @@ TensorMechanicsApp::associateSyntax(Syntax & syntax, ActionFactory & action_fact
   syntax.registerActionSyntax("PressureAction", "BCs/Pressure/*");
 
   syntax.registerActionSyntax("GeneralizedPlaneStrainAction", "Modules/TensorMechanics/GeneralizedPlaneStrain/*");
+  syntax.registerActionSyntax("CommonTensorMechanicsAction", "Modules/TensorMechanics/Master");
```

Comment:
Couldn't you just register `TensorMechanicsActionBase` here and get rid of `CommonTensorMechanicsAction`?